schedule/yast/autoyast/autoyast_btrfs_quota.yaml

```diff
@@ -0,0 +1,29 @@
+---
+name: autoyast_btrfs_quota
+description: >
+  AutoYaST installation with BTRFS subvolume quotas. Verifies quota limit on the installed system
+  and if its size corresponds to the configured quota limit during installation.
+vars:
+  AUTOYAST_PREPARE_PROFILE: 1
+  DESKTOP: textmode
+schedule:
+  - autoyast/prepare_profile
+  - installation/bootloader_start
+  - autoyast/installation
+  - autoyast/console
+  - autoyast/login
+  - autoyast/verify_btrfs_quota
```

Comment:
What's about validation of the cloned profile?

Reply:
Done. Waiting for VRs to be finished.